### PR TITLE
[INOYI-1]: Remove deprecated entity.query service from Drupal\openy_r…

### PR DESCRIPTION
**Pull request by AleksandrPrykhodkoDev**
Original source:
https://github.com/ymcatwincities/openy/issues/1421

## Steps for review

- [x] Check code changes
- [ ] Check Schedules paragraph after refactoring

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/custom/openy_repeat/src/RepeatManager.php [hunk 1]
```diff
@@ -75,8 +68,7 @@ class RepeatManager implements SessionInstanceManagerInterface {
   /**
    * Constructor.
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager, QueryFactory $entity_query, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
-    $this->entityQuery = $entity_query;
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
```
Comment (dchubar-fj) [inline modules/custom/openy_repeat/src/RepeatManager.php]: @AleksandrPrykhodkoDev you have removed the entity query dependency from the constructor, but I don't see where did you update the service definition for the class?

File: modules/custom/openy_session_instance/src/SessionInstanceManager.php [hunk 2]
```diff
@@ -67,8 +60,7 @@ class SessionInstanceManager implements SessionInstanceManagerInterface {
   /**
    * Constructor.
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager, QueryFactory $entity_query, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
-    $this->entityQuery = $entity_query;
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
```
Comment (dchubar-fj) [inline modules/custom/openy_session_instance/src/SessionInstanceManager.php]: Usage of `$this->entityQuery` should be replaced with `$this->entityTypeManager` as well. The class is broken now.

**Review by dchubar-fj — CHANGES_REQUESTED** [review]

**Review by podarok — DISMISSED** [review]

**Comment by podarok** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2123/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2123/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2123/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2123/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2123/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2123/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2123/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2123/

File: modules/custom/openy_repeat/src/RepeatManager.php [hunk 3]
```diff
@@ -72,29 +67,53 @@ class RepeatManager implements SessionInstanceManagerInterface {
...
     $this->configFactory = $configFactory;
     $this->config = $this->configFactory->get('openy_repeat.settings');
+    $this->moduleHandler = $module_handler;
+    $this->moderationWrapper = $moderationWrapper;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
```
Comment (dchubar-fj) [inline modules/custom/openy_repeat/src/RepeatManager.php]: @AleksandrPrykhodkoDev this service is defined in the YML file, so why do you use this approach here? It doesn't work, the service is broken now. Please define all dependencies in `openy_session_instance.services.yml` and update the original `SessionInstanceManager.php` service for that.

**Review by dchubar-fj — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2124/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2124/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2124/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2124/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2125/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2125/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2125/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2125/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2125/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2125/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2125/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2126/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2126/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2125/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2126/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2125/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2126/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2175/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2175/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2175/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2214/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2214/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2214/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2214/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2214/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2214/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2214/blogWCAG2AAhtmlcs.txt

**Review by podarok — APPROVED** [review]

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2214/

**Pull request merged**